browser/ui/BUILD.gn

```diff
@@ -609,6 +612,10 @@ source_set("ui") {
     ]
   }
 
+  if (enable_request_otr) {
```

Comment:
I think you have this backwards, the buildflag dep should not be behind the buildflag, but browser and common above should be

Reply:
fixed